### DISCUSSION TITLE
Breakpoints not working with Next.js

### DISCUSSION THREAD

**Discussion by secondl1ght** [tailwindlabs/tailwindcss#11983] [Help]
I have a fresh Next.js app setup with Tailwindcss and everything appears to be working EXCEPT for breakpoints. I have done some debugging but I didn't miss any setup steps in the Tailwind docs https://tailwindcss.com/docs/guides/nextjs. What am I missing here...? 🤔

**Comment by wongjn** [comment]
Consider checking:
- The `content` file globs sufficiently cover the file that the code appears in.
- [String concatenation/interpolation is not used to build the class name](https://tailwindcss.com/docs/content-configuration#dynamic-class-names). 
- The class name would resolve to a Tailwind class that Tailwind would generate a rule for.

Otherwise, consider providing a remotely-hosted git repository that reproduces the unexpected behavior.

**Reply by secondl1ght** [reply]
OK all of these are true, so I will create a reproduction to share.
